apple/LayoutReanimation/REAAnimationsManager.m

```diff
@@ -595,6 +606,15 @@ - (BOOL)hasAnimationForTag:(NSNumber *)tag type:(LayoutAnimationType)type
   return _hasAnimationForTag(tag, type);
 }
 
+- (BOOL)isDisabledExiting:(NSNumber *)tag
+{
+  if (!_isDisabledExiting) {
+    // It can happen during reload.
+    return NO;
+  }
+  return _isDisabledExiting(tag);
+}
```

Comment:
```suggestion
- (BOOL)isExitingDisabled:(NSNumber *)tag
{
  if (!_isExitingDisabled) {
    // It can happen during reload.
    return NO;
  }
  return _isExitingDisabled(tag);
}
```